blog/2023-10-29-service-exceptions-and-translations.md

```diff
@@ -0,0 +1,21 @@
+---
+author: Jan Bouwhuis
+authorURL: https://github.com/jbouwh
+authorImageURL: https://avatars.githubusercontent.com/u/7188918?s=96&v=4
+title: Exception handling during and translation support
+---
+
+## Exception handling during service calls
+
+When an exception is raised during a service call and the printing of stack traces is not warranted, then this can cause that the logs are being spammed. This happens when validation violations cause a service call to fail and we can print a clear error message but do not need a stack trace to make thing clear.
```

Comment:
Please start with describing the current situation, then describe how we want to change the situation and finally  describe the new feature that will allow the change in behavior.

Short example:
Currently service calls that raise exceptions will log full stack traces. Service calls that fail due to invalid user input don't need a stack trace but would benefit from a helpful error message in the users own language.

To be able to suppress the stack trace in these cases...